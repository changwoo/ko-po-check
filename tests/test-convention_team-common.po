# -*-text-*-
# Some guys want to express their politics using header...
msgid ""
msgstr ""
"Project-Id-Version: test 14\n"
"POT-Creation-Date: 2010-03-03 00:16+0900\n"
"PO-Revision-Date: 2010-03-03 00:16+0900\n"
"Last-Translator: Blah Blah <cwryu@debian.org>\n"
"Language-Team: Dokdo in Korea <blah@blah.blah>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

msgid ""
msgstr ""
"Project-Id-Version: test 14\n"
"POT-Creation-Date: 2010-03-03 00:16+0900\n"
"PO-Revision-Date: 2010-03-03 00:16+0900\n"
"Last-Translator: Blah Blah <blah@blah.blah>\n"
"Language-Team: <Dokdo@korea.land>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# <Dokdo@korea.land>
msgid ""
msgstr ""
"Project-Id-Version: test 14\n"
"POT-Creation-Date: 2010-03-03 00:16+0900\n"
"PO-Revision-Date: 2010-03-03 00:16+0900\n"
"Last-Translator: Blah Blah <blah@blah.blah>\n"
"Language-Team: <blah@blah.blah>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
