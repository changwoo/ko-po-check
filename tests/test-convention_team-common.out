-:4: 정치적 문구를 쓰는 곳이 아닙니다: Dokdo in Korea <blah@blah.blah>
-:15: 정치적 문구를 쓰는 곳이 아닙니다: <Dokdo@korea.land>
-:27: 정치적 문구를 쓰는 곳이 아닙니다: <Dokdo@korea.land>
